documentation/modules/proc-view-connections-qdstat.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// assemblies/assembly-qdstat.adoc
+
+[id='view-connections-qdstat-{context}']
+= Viewing router connections using qdstat
+
+A connection to the container running `qdstat` is created by default on the router. You can view the router connections using `qdstat`.
```

Comment:
As the use case here is to inspect the state of the router I would say we can leave out the explanation of the connection to the container running qdstat.

> You can view the router connections using `qdstat`.